alainmateso/iFind
# CH Create Seeders for Admins and Categories

#### What does this PR do?

It adds seeder files for Admins and Category

#### Description of Task to be completed?

To create a default admin that creates other admins and users
To create a list of categories that a user can select from when creating categories

#### How should this be manually tested?

1. Download the repo.
2. Run ` npm install ` to install dependancies
2. The seeders take the default values from the ` .env ` files.

Add the following to your .env file.
```javascript

ADMIN_FIRSTNAME=
ADMIN_LASTNAME=
ADMIN_PHONENUMBER=
ADMIN_EMAIL=

```

The default password is ` "default" `
4. Run `  sequelize db:seed:all ` to add seeds to your tables 
5. Check postgres DB (users & categories) to ensure seeds are in the tables.

#### Any background context you want to provide?

Due to difficulty in creating posts and items, a default admin and categories was necessary. 

#### What are the relevant pivotal tracker stories?
##### #168562201

#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
